File: java/Kode program/BrandInfo.java
public class BrandInfo {
    private String brandName;
    private String originCountry;
    private int establishedYear;

    public void setBrandName(String b) { brandName = b; }
    public String getBrandName() { return brandName; }

    public void setOriginCountry(String c) { originCountry = c; }
    public String getOriginCountry() { return originCountry; }

    public void setEstablishedYear(int y) { establishedYear = y; }
    public int getEstablishedYear() { return establishedYear; }

    public void printBrand() {
        System.out.println("Brand: " + brandName);
        System.out.println("Origin Country: " + originCountry);
        System.out.println("Established Year: " + establishedYear);
    }
}

File: python/Kode program/BrandInfo.py
class BrandInfo:
    def __init__(self):
        self.brandName = ""
        self.originCountry = ""
        self.establishedYear = 0

    def setBrandName(self, bn):
        self.brandName = bn

    def getBrandName(self):
        return self.brandName

    def setOriginCountry(self, oc):
        self.originCountry = oc

    def getOriginCountry(self):
        return self.originCountry

    def setEstablishedYear(self, ey):
        self.establishedYear = ey

    def getEstablishedYear(self):
        return self.establishedYear

    def printBrand(self):
        print(f"Brand: {self.brandName}")
        print(f"Origin Country: {self.originCountry}")
        print(f"Established Year: {self.establishedYear}")
